score-xml/src/test/java/io/cloudslang/content/xml/actions/AddAttributeTest.java

```diff
@@ -0,0 +1,67 @@
+package io.cloudslang.content.xml.actions;
+
+import io.cloudslang.content.xml.utils.Constants;
+import org.junit.Assert;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URI;
+import java.util.Map;
+
+/**
+ * Created by markowis on 25/02/2016.
+ */
+public class AddAttributeTest {
+
+    @Test
+    public void testAddAttribute() throws Exception{
+        AddAttribute toTest = new AddAttribute();
+
+        URI resource = getClass().getResource("/xml/test.xml").toURI();
+        String xml = FileUtils.readFileToString(new File(resource));
```

Comment:
move this common part into a setUp method annotated with @ Before, see other tests on how to do this